data/json/mapgen_palettes/wasp_palette.json

```diff
@@ -0,0 +1,144 @@
+[
+  {
+    "type": "palette",
+    "id": "wasp_palette",
+    "terrain": {
+      "x": "t_paper_brood",
+      "X": "t_paper_hard",
+      "_": "t_floor_paper_hard",
+      "*": "t_roof_paper_hard",
+      ".": "t_floor_paper_noroof",
+      "8": "t_floor_paper_noroof",
+      "R": "t_radio_tower",
+      "a": "t_railing",
+      ",": "t_metal_floor_no_roof",
+      "<": "t_stairs_up",
+      ">": "t_stairs_down",
+      ":": "t_null",
+      "#": "t_metal_floor_no_roof"
+    },
+    "monster": { "8": { "group": "GROUP_WASP_QUEEN" } },
+    "furniture": { ":": "f_cellphone_booster", "#": "f_small_satelitte_dish" },
+    "nested": { "O": { "chunks": [ "wasp_brood" ] } }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "wasp_brood",
+    "//": "Empty cell",
+    "weight": 100,
+    "object": { "mapgensize": [ 1, 1 ], "rows": [ "0" ], "terrain": { "0": "t_open_air" } }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "wasp_brood",
+    "//": "Egg cell.",
+    "weight": 200,
+    "object": {
+      "mapgensize": [ 1, 1 ],
+      "rows": [ "0" ],
+      "terrain": { "0": "t_floor_brood_fake" },
+      "item": { "0": { "item": "egg_wasp" } }
```

Comment:
Did you define the item `egg_wasp`?

Reply:
Only in the other wasp PR. I'm a bit paranoid about accidentally overwriting some stuff if I decide to fiddle with them on the other branch.